app/src/main/java/ru/netology/nmedia/viewmodel/PostViewModel.kt

```diff
@@ -0,0 +1,13 @@
+package ru.netology.nmedia.viewmodel
+
+import androidx.lifecycle.ViewModel
+import ru.netology.nmedia.repository.PostRepository
+import ru.netology.nmedia.repository.PostRepositoryInMemory
+
+class PostViewModel: ViewModel() {
+    private val repository:PostRepository = PostRepositoryInMemory()
+    val data = repository.get()
```

Comment:
В данном вариант сгенерируется примерно следующий код:
```kotlin
private val _data = repository.get()
fun getData() = _data
```

Если заменить на вариант
```suggestion
    val data get() = repository.get()
```
то сгенерируется только `fun getData() = repositor.get()`, что производительней, в два раза меньше и лучше отражает наши намерения.

P.S.: Если в `repository` замените функцию на на свойство, то можно будет даже заменить на следующую запись.
```suggestion
    val data by repository::data
```
Я это не часто практикую. Может быть с функцией тоже сработает - проверьте.